apps/meteor/client/methods/unpinMessage.ts

```diff
@@ -26,7 +26,7 @@ Meteor.methods<ServerMethods>({
 			return 0;
 		}
 		dispatchToastMessage({ type: 'success', message: t('Message_has_been_unpinned') });
-		return ChatMessage.update(
+		ChatMessage.update(
```

Comment:
needed?

Reply:
yes.. this is client stub version that should reflect same as server version